drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -620,6 +674,224 @@ def self.sections(report) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticC
+              HudUtility2024.performance_reporting[:sh],
+              HudUtility2024.performance_reporting[:oph],
+              HudUtility2024.performance_reporting[:rrh],
+            ].flatten.include?(item.project_type)
+          },
+          limiter: ->(item) {
+            return false unless hoh?(item) || item.veteran
+            return false unless [
+              HudUtility2024.performance_reporting[:es_entry_exit],
+              HudUtility2024.performance_reporting[:th],
+              HudUtility2024.performance_reporting[:psh],
+              HudUtility2024.performance_reporting[:so],
+              HudUtility2024.performance_reporting[:sh],
+              HudUtility2024.performance_reporting[:oph],
+              HudUtility2024.performance_reporting[:rrh],
+            ].flatten.include?(item.project_type)
+
+            item.vamc_station.blank?
+          },
+        },
+        entering_from_streets_issues: {
+          title: 'Client Entering From Streets, ES, or SH',
+          description: 'Prior living situation is "Data not collected" (99) or blank',
+          required_for: 'Adults and HoH',
+          detail_columns: default_detail_columns + [
+            :living_situation,
+          ],
+          denominator: ->(item) { hoh_or_adult?(item) },
+          limiter: ->(item) {
+            return false unless hoh_or_adult?(item)
+
+            item.living_situation.blank? || item.living_situation == 99
+          },
+        },
+        homeless_living_situation_issues: {
+          title: 'HP - Homeless Living Situation',
+          description: 'HP adult has a homeless prior living situation',
+          required_for: 'Adults in HP at entry',
+          detail_columns: default_detail_columns + [
+            :living_situation,
+          ],
+          denominator: ->(item) { adult?(item) && HudUtility2024.performance_reporting[:prevention].include?(item.project_type) },
+          limiter: ->(item) {
+            return false unless adult?(item)
+            return false unless HudUtility2024.performance_reporting[:prevention].include?(item.project_type)
+
+            HudUtility2024.homeless_situations(as: :prior).include?(item.living_situation)
+          },
+        },
+        non_homeless_living_situation_issues: {
+          title: 'RRH - Non-Homeless Living Situation',
+          description: 'RRH adult has a non-homeless prior living situation',
+          required_for: 'Adults in RRH at entry',
+          detail_columns: default_detail_columns + [
+            :living_situation,
+          ],
+          denominator: ->(item) { adult?(item) && HudUtility2024.performance_reporting[:rrh].include?(item.project_type) },
+          limiter: ->(item) {
+            return false unless adult?(item)
+            return false unless HudUtility2024.performance_reporting[:rrh].include?(item.project_type)
+
+            HudUtility2024.situation_type(item.living_situation) == 'Homeless'
```

Comment:
For consistency, isn't this the same (and similar to the HP check?)
```suggestion
            ! HudUtility2024.homeless_situations(as: :prior).include?(item.living_situation)
```